packages/inertia/src/inertia.js

```diff
@@ -1,8 +1,10 @@
 import Axios from 'axios'
 import Modal from './modal'
+import Debounce from './debounce'
 import Progress from './progress'
 
 export default {
+  saveScrollPositions: null,
```

Comment:
Not 100% sure, but I _think_ you could declare the function here already.

```suggestion
  saveScrollPositions: debounce(() => { /* … */ }),
```

Reply:
No, that won't work, because there's a reference to the current object in it that will fail otherwise:
https://github.com/inertiajs/inertia/blob/5b9eab2e00db8b7b7f17d9b3d3497ee0297373bb/packages/inertia/src/inertia.js#L29-L31

Reply:
I tried that originally, but got this error:

```
inertia.js?8c02:8 Uncaught TypeError: Cannot read property 'setState' of undefined
    at Object.r (inertia.js?8c02:8)
    at eval (debounce.js?1b85:5)
```